### PR TITLE
Fix version comparison to work with 1.10.0.beta.1

### PR DESCRIPTION
**Pull request by dnkoutso**
closes https://github.com/square/cocoapods-generate/issues/68

### CODE REVIEW

**Comment by dnkoutso** [comment]
Verified this works with 1.10.0.beta.1

**Comment by segiddins** [comment]
Please bundle update CocoaPods to verify tests continue to pass, and then bump the version

File: Gemfile [hunk 1]
```diff
@@ -6,9 +6,8 @@ git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
 gemspec
 
-gem 'activesupport', '4.2.11.3'
```
Comment (dnkoutso) [inline Gemfile:9]: removed this as cocoapods now provides `> 5`

I figured we dont explicitly care for it anyway.

**Comment by dnkoutso** [comment]
cocoapods updated and fixed specs as well as updated integration specs

File: spec/cocoapods/command/gen_spec.rb [hunk 2]
```diff
@@ -47,6 +47,7 @@
 
               Options:
 
+                  --allow-root                           Allows CocoaPods to run as root
```
Comment (justinseanmartin) [inline spec/cocoapods/command/gen_spec.rb:50]: Is this just an undocumented setting, or where is this setting coming from?

Comment (dnkoutso) [reply spec/cocoapods/command/gen_spec.rb:50]: its from 1.10.0.beta release with a new option to run CocoaPods as root. The bump in this PR caused this test to fail.

**Comment by paulb777** [comment]
@dnkoutso With 1.10.0 maturing, it would be great to get this in a 2.1.0

**Comment by ilmat192** [comment]
Since CocoaPods 1.10 has been [released](https://github.com/CocoaPods/CocoaPods/releases/tag/1.10.0), are there any plans to release an update for `cocoapods-generate` with this fix?

**Comment by chris-hatton** [comment]
Also keenly interested in this fix being released: this is critical for using Cocoapods-Generate with latest versions of Xcode, since there's a [Cocoapods bug](https://github.com/CocoaPods/CocoaPods/pull/9905) fixed in 1.10 for Xcode >= 12 compatibility.  Currently this is blocking my use-case completely.

**Comment by segiddins** [comment]
2.0.1 released

**Pull request merged**